### PR TITLE
Add generator for regular expressions

### PR DESCRIPTION
**Pull request by Cka3o4Huk**
Hi,

This patch introduces generator for regular expressions. It allows to cover plenty of cases like email generation. Generator works like wrapper of generix library, so patch is very simple. 

Please review it and merge if OK.

Thank you in advance!

P.S. could you push library to Maven Central?

### CODE REVIEW

File: src/main/java/com/narmnevis/range/generator/GenerexGenerator.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package com.narmnevis.range.generator;
+
+import com.mifmif.common.regex.Generex;
+import com.narmnevis.range.Generator;
+import com.narmnevis.range.RangeContext;
+
+public class GenerexGenerator implements Generator {
+
+	private Generex generex;
```
Comment (nobeh) [inline src/main/java/com/narmnevis/range/generator/GenerexGenerator.java]: Could you please add `final`?

**Comment by nobeh** [comment]
Hi,

Thanks for the PR. It's actually very useful and well though of.

Cheers,
Behrooz

File: src/main/java/com/narmnevis/range/generator/GenerexGenerator.java (hunk 1 above)
Comment (Cka3o4Huk) [inline src/main/java/com/narmnevis/range/generator/GenerexGenerator.java]: Done. PR's updated.

**Comment by nobeh** [comment]
Also created #8. Thanks.

**Pull request merged**